packages/playwright-test/src/webServer.ts

```diff
@@ -122,13 +122,16 @@ async function isPortUsed(port: number): Promise<boolean> {
   return await innerIsPortUsed('127.0.0.1') || await innerIsPortUsed('::1');
 }
 
-async function isURLAvailable(url: URL) {
+async function isURLAvailable(url: URL, ignoreHTTPSErrors?: boolean) {
   return new Promise<boolean>(resolve => {
-    (url.protocol === 'https:' ? https : http).get(url, res => {
+    (url.protocol === 'https:' ? https : http).get(url, {
+      rejectUnauthorized: !ignoreHTTPSErrors,
```

Comment:
in theory we only need to pass it when its 'http:' protocol, but probably still fine.